process_onestop_sr_report/preprocessing.py

```diff
@@ -54,6 +55,12 @@ class ArgsParser(Tap):
     """
 
     filter_query: str = ""
+    
+    """
+    NOTE: To extract surprisal from state-spaces/mamba-* variants, better to first run:
```

Comment:
What do you mean by 'better to'? maybe -
```suggestion
    NOTE: To extract surprisal from state-spaces/mamba-* variants first run:
```